cluster/gce/util.sh

```diff
@@ -2706,6 +2738,8 @@ function kube-up() {
       check-cluster
       validate-cluster-status
       create-node-port
+      # create arktos network crd for scale-up
+      "${KUBE_ROOT}/cluster/kubectl.sh" --kubeconfig="$HOME/.kube/config" apply -f "${KUBE_ROOT}/pkg/controller/artifacts/crd-network.yaml"
```

Comment:
this creation of netword CRD should already have been in pod code branch.

Reply:
Ok, I can remove it.

Reply:
done.